ex01.exs

```diff
@@ -27,9 +27,31 @@ defmodule Ex01 do
           blank lines, vertical alignment
   """
   
-  def counter(value \\ 0) do
```

Comment:
## -20

This code doesn't work for a couple of reasons:

1. You created a second Ex01 module inside the top-level one, so confusing the location of the `counter` function.

2. Your response from `next_value` is

  ~~~ elixir
  send from, {"next_is", value}
  ~~~

  You're sending back a string, but the rest of the code assumes the symbol `:next_is`. As a result, your receives never work, and the vounter receive block times out.